crates/biome_js_formatter/src/comments.rs

```diff
@@ -301,6 +302,75 @@ fn handle_labelled_statement_comment(
     }
 }
 
+/// Moves comments in declaration statements to behind the identifier
+///
+/// ```javascript
+/// declare module /* comment */ A {}
+/// declare /* module */ global {}
+/// ```
+fn handle_declare_comment(comment: DecoratedComment<JsLanguage>) -> CommentPlacement<JsLanguage> {
+    match (comment.enclosing_node().kind(), comment.following_node()) {
+        // Check if it is a declare statement
+        (JsSyntaxKind::TS_DECLARE_STATEMENT, Some(following)) => {
+            println!("following: {:?}", following);
```

Comment:
To remove

Reply:
Whoops, that was me debugging my code, let me remove that now.